pkg/leaderelection/leaderelection.go

```diff
@@ -2,18 +2,14 @@ package leaderelection
 
 import (
 	"context"
+	"sync"
 	"time"
 
 	"github.com/razorpay/metro/pkg/logger"
 	"github.com/razorpay/metro/pkg/registry"
 	"k8s.io/apimachinery/pkg/util/wait"
 )
 
-const (
-	// JitterFactor used in wait utils
-	JitterFactor = 1.2
-)
-
 // NewLeaderElector creates a LeaderElector from a LeaderElection Config
 func NewLeaderElector(c Config, registry registry.IRegistry) (*LeaderElector, error) {
```

Comment:
as per go convention packageName.New() is preffered over packageName.NewPackageName()